src/tasks/AppleAppBuilder/AppleAppBuilder.cs

```diff
@@ -12,11 +12,44 @@
 
 public class AppleAppBuilderTask : Task
 {
+    private string targetOS = TargetNames.iOS;
+
     /// <summary>
     /// The Apple OS we are targeting (iOS or tvOS)
     /// </summary>
     [Required]
-    public string TargetOS { get; set; } = TargetNames.iOS;
+    public string TargetOS
+    {
+        get
+        {
+            return targetOS;
+        }
+
+        set
+        {
+            switch (value.ToLower())
+            {
+                case TargetNames.iOS:
```

Comment:
Make sure this still works.  I don't know if xcode is particular w/ case or not.

Reply:
Right. The current proposed fix seems to be equivalent to just doing:

`targetOS = value.ToLower()` ? It seems like if this is what we need the switch wouldn't be needed at all?

Reply:
Addressed, thanks